File: 1. Conditional Problems/Condional Problems in C/p18.c
// write a program that takes an integer number as input, 
// which is greater than 99 and less than 1000. 
// Print the first digit of the number. As an example, if the input number is 253, print 2.

#include<stdio.h>

int main() {
    int n;
    printf("Take a Number between 99 and 1000: ");
    scanf("%d",&n);

    if (n<99 || n >1000) {
        printf("Invalid Input");
    }
    while (n>9){
        n /= 10;
        
    }
    printf("First Digit is: %d",n);
}

File: 1. Conditional Problems/Conditional Problems in Python/p18.py
# Input an integer number
n = int(input("Enter a number between 99 and 1000: "))

# Check if the number is in the valid range
if n <= 99 or n >= 1000:
    print("Invalid Input")
else:
    # Extract the first digit by continuously dividing by 10
    while n > 9:
        n //= 10  # Use floor division to truncate the number
    
    print(f"First Digit is: {n}")
